utils/check_copies.py

```diff
@@ -197,7 +197,7 @@ def get_model_list():
     # If the introduction or the conclusion of the list change, the prompts may need to be updated.
     _start_prompt = "🤗 Transformers currently provides the following architectures"
     _end_prompt = "1. Want to contribute a new model?"
-    with open(os.path.join(REPO_PATH, "README.md"), "r", encoding="utf-8", newline="\n") as f:
+    with open(os.path.join(REPO_PATH, "README.md"), "r", encoding="utf-8", newline=None) as f:
```

Comment:
@sgugger - did we have problems with that on Windows before?

Reply:
This was specifically changed for Windows support and should probably be left as is along with following the instructions [here](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#develop-on-windows).

Reply:
@sgugger I see. I think the issue then is that this standardizes the writing of python files to use LF, but `shutil.copy` used in [this test](https://github.com/huggingface/transformers/blob/master/tests/test_utils_check_copies.py#L58), copies using CRLF endings. Would it be ok to use another method of copying in this test, such that the written files use LF.

Reply:
As long as this other method works on other OS I'm fine with it.